src/nxdoc/nuxeo-server/installation/docker-image.md

```diff
@@ -119,7 +117,7 @@ docker run --name nuxeo \
 
 #### NUXEO_PACKAGES
 
-`NUXEO_PACKAGES` allows to define a space separated list of Nuxeo packages to install at startup.
+`NUXEO_PACKAGES` allows to define a space separated list of Nuxeo packages to install at startup. 
```

Comment:
useless

Reply:
I don't get it

Reply:
The extra space at the end of the sentence.